Neos.Fusion/Classes/FusionObjects/TagImplementation.php

```diff
@@ -76,7 +76,7 @@ protected function getContent()
 
     /**
      * The tag attributes dataStructure
-     * If anything but an array is returned the value is casted to string
+     * If anything but an iterable is returned the value is casted to string
```

Comment:
Does that comment even make sense (any longer) and shouldn't it be
```
@return iterable|string
```
?

Reply:
Not sure, fusion usually behaves very relaxed and does not throw type errors when it has a chance to do anything else. 

With the current implementation anything returned from attributes will be cast to string. By not enforcing any type thus change keeps that behavior and adds a new path for iterables as new default.

So i would not enforce strict types here.

Reply:
Forget my first remark, and the second was just about the return annotation: